aws/aws-toolkit-jetbrains
# Mark the SAM build directory as excluded

## Types of changes


## Testing
1. Make a project via wizard
1. Verify that folder is excluded in project structure
1. Deploy an app
1. Verify that the directory is excluded in the project view

## Screenshots (if appropriate)

## Checklist



## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.
